.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
README file should have a project name, description about project features, and instructions for users to set it up locally. Moreover, please remove unnecessary details like Author2. Kindly remember that README is the first point of interaction for your project so it should be clean and professional!